crates/webidl/src/idl_type.rs

```diff
@@ -641,9 +752,9 @@ impl<'a> IdlType<'a> {
                 }
             }
 
-            IdlType::Any => Ok(js_value),
-            IdlType::Void => Ok(None),
-            IdlType::Callback => Ok(js_sys("Function")),
+            IdlType::Any => Ok(js_value.honestly()),
+            IdlType::Void => Ok(None.honestly()), // FIXME what does it mean?
+            IdlType::Callback => Ok(js_sys("Function").honestly()), // FIXME can we give type?
```

Comment:
I checked around a bit but indeed, the information is long gone by this time. Added a generic confession anyway for consistency (as it's another case of "just because the signature says Function, there's actually more data but we don't express it") -- sure it'd be nice to have a better one here, but that's about as far as I can take it now.

(If you consider the added texts extraneous, I can make the confession not emit any text -- but for future expansion it's probably good to have it visible in the code).